README.md

```diff
@@ -8,6 +8,28 @@ Clementine is a modern music player and library organizer for Windows, Linux and
 - Buildbot: http://buildbot.clementine-player.org/grid
 - Latest developer builds: http://builds.clementine-player.org/
 
+Opening an issue
+----------------
+### Ask a new feature
+
+Please :
+
+ * Check if the new feature is not already implemented (Changelog)
+ * Check if another person didn't already open an issue
+ * If there is already an opened issue there is no need to comment "+1", it won't help :o)
+
+### Report a bug
+
+Please :
+ 
+ * Try with the latest developer build to see if the bug is still present. If if works like a charm although you see an opened issue please comment it to explain the issue was fixed
```

Comment:
Yes. Or the database where all info about songs are indexed. I think this happened once or twice in the past.
I don't think we ever had anything more serious, like deleting songs by accident.
But there is also the problem that, in the case of some db schema change for example, this is not backward compatible, so users cannot downgrade Clementine after IIRC.
But well, this doesn't happen very often, so just warn "this might break few things like users settings" might be enough.
Actually, the main problem I think users might have when trying developers builds is that Clementine wouldn't start because a dependency isn't indicated, a DLL is missing, etc. or crash somewhere else, so it's not a big deal: in this case they can safely reinstall the last stable one without any problem.